spec/spec_helper.rb

```diff
@@ -13,4 +13,5 @@
 begin
   require 'spec_helper_local'
 rescue LoadError
+  puts 'Rescue Load Error'
 end
```

Comment:
All changes in this file should probably be done in Module Sync rather than in the actual file.

Reply:
Didn't know that.
Was brought up as a rubocop error so I changed it.

Reply:
Fix has been made to modulesync